cmd/status.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/codegangsta/cli"
+	"github.com/exercism/cli/api"
+	"github.com/exercism/cli/config"
+)
+
+// Status command allows a user to view their progress in a given language track
+func Status(ctx *cli.Context) {
+	c, err := config.New(ctx.GlobalString("config"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	args := ctx.Args()
+
+	if len(args) != 1 {
+		msg := "Usage: exercism status LANGUAGE"
+		log.Fatal(msg)
+	}
+
+	language := args[0]
+	client := api.NewClient(c)
+	status, err := client.Status(language)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	printResponse(language, status)
+}
+
+func printResponse(language string, status [][]string) {
+	recent := status[0]
+	fetched := status[1]
+	skipped := status[2]
```

Comment:
Yeah, using a custom struct type for status will make this all much simpler to deal with.